utils/help_commands.py

```diff
@@ -0,0 +1,67 @@
+"""
+Help message of each commands
+"""
+
+# Discord
+import discord
+
+from utils.embed_object import EmbedData
+from utils.utils import create_embed
+
+
+def create_help_title_desc(bot, name, summoner_name: bool):
+    """
+    Make global title and description variables for help embed message
+    """
+    # pylint: disable=global-variable-undefined, invalid-name
+    global help_title
+    global help_description
+    help_title = "How to use {} command".format(name)
+
+    if summoner_name:
+        help_description = "** **\n<@!{0}> **{1} summoner_name**".format(
+            bot.user.id, name
+        )
+    else:
+        help_description = "** **\n<@!{0}> **{1}**".format(bot.user.id, name)
+
+
+def create_help_fields(name: list):
+    """
+    Make a global value variable for help embed message
+    """
+    # pylint: disable=global-variable-undefined, invalid-name
+    global help_information
+    help_information = ""
+    for desc in name:
+        help_information += desc + "\n\n"
+
+
+def create_help_embed():
+    """create embed for help message"""
+    try:
+        embed_data = EmbedData()
+
+        embed_data.title = help_title
+
```

Comment:
You have three functions that are tightly coupled to each other.. What if I call `create_help_fields()` before calling `create_help_title_desc()`??\
If you wana do something like this, you gotta create class where each class functions will reference the class variables. Otherwise make these independent

Reply:
순서가 바껴도 괜찮긴한데, 만약 둘 중 하나라도 입력을 안하면 버그가 생기네요ㅠ 
예를들어 `create_help_title_desc()`는 호출했는데 `create_help_fields()`를 호출 안 하면, 
global 변수로 설정해놨기때문에 그 전 `command` 에 대한 `description` 이 나오네요.. 
수정하겠습니다!